src/uiSelectChoicesDirective.js

```diff
@@ -75,12 +73,24 @@ uis.directive('uiSelectChoices',
           }
         });
 
-        attrs.$observe('refreshDelay', function() {
+        attrs.$observe('refreshDelay', function(refreshDelay) {
           // $eval() is needed otherwise we get a string instead of a number
-          var refreshDelay = scope.$eval(attrs.refreshDelay);
+          refreshDelay = scope.$eval(refreshDelay);
           $select.refreshDelay = refreshDelay !== undefined ? refreshDelay : uiSelectConfig.refreshDelay;
         });
 
+        attrs.$observe('nullValue', function(value) {
+          value = value ? scope.$eval(value) : undefined;
+          if (value === null || value !== undefined) {
+            $select.nullValue = value;
```

Comment:
This updates the `nullValue` on the controller but doesn't update the null value in the `items` list.
If we decide to ignore updates to the `nullValue` expression then we shouldn't be using `attrs.$observe`.

Reply:
Added refreshItems call